iphone/Classes/TiUIScrollableViewProxy.m

```diff
@@ -336,13 +336,28 @@ -(CGFloat)autoHeightForSize:(CGSize)size
     return result;
 }
 
-- (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id <UIViewControllerTransitionCoordinator>)coordinator
 {
     if ([self viewAttached]) {
         [(TiUIScrollableView*)[self view] manageRotation];
     }
 }
 
+- (void)systemLayoutFittingSizeDidChangeForChildContentContainer:(id <UIContentContainer>)container
+{
+    
+}
+
+- (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withTransitionCoordinator:(id <UIViewControllerTransitionCoordinator>)coordinator
+{
+    
+}
+
+- (void)preferredContentSizeDidChangeForChildContentContainer:(id <UIContentContainer>)container
+{
+    
+}
```

Comment:
This is intentional. No need to call super. These are end calls . These protocols  are implemented here  to play safe. If it get call from somewhere without check, app will ensure to not crash.  Required changes for scrollview is implemented as per previous implementation , in method -
- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id <UIViewControllerTransitionCoordinator>)coordinator